types/react-places-autocomplete/index.d.ts

```diff
@@ -53,24 +53,24 @@ export interface PropTypes {
             role: 'combobox';
             'aria-autocomplete': 'list';
             'aria-expanded': boolean;
-            'aria-activedescendant': string | null;
+            'aria-activedescendant': string | undefined;
             disabled: boolean;
             onKeyDown: React.KeyboardEventHandler;
-            onBlur: () => void;
+            onBlur: React.FocusEventHandler;
             value: string | undefined;
             onChange: (ev: { target: { value: string }}) => void;
         } & InputProps;
         getSuggestionItemProps: <SuggestionProps extends {}>(suggestion: Suggestion, options?: SuggestionProps) => {
             key: number;
-            id: string | null;
+            id: string | undefined;
```

Comment:
```suggestion
            id?: string;
```

Reply:
Yeah, I wanted to do the same at first but the code already uses `| undefined` for other attributes.
And I didn't want the diff to be bigger.

Anyway, `| undefined` or `?` is just a matter of taste.

Reply:
Ah, got it. No worries … it just was my recommendation but it's totally optional. :)